suitcase/tests/test_spec.py

```diff
@@ -183,11 +183,11 @@ def test_round_trip_from_specfile(spec_filename, db_all):
 
 @pytest.mark.skipif(sys.version_info == (2,7),
                     reason="bluesky related tests need python 3.5, 3.6")
-def test_round_trip_from_run_engine(db_all, RE):
-    import bluesky
-    # generate a new specfile
+def test_round_trip_from_run_engine(db_all):
```

Comment:
Why not use the `RE` test fixture here?

Reply:
RE is only used in this function. It is easy to make pytest skip this test function for py27. There can be other ways.

Reply:
I suspect you will want to use it in other functions, though. See my comment about temperature_ramp.